tools/ci_build/github/apple/onnxruntime-mobile-c.podspec.template

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |spec|
+    spec.name                   = "onnxruntime-mobile-c"
+    spec.version                = "${ORT_VERSION}"
+    spec.authors                = { "ONNX Runtime" => "onnxruntime@microsoft.com" }
+    spec.license                = { :type => "MIT" }
+    spec.homepage               = "https://github.com/microsoft/onnxruntime"
+    spec.source                 = { :http => "_ORT_DOWNLOAD_URL_" }
```

Comment:
>_ORT_DOWNLOAD_URL_ [](http://example.com/codeflow?start=46&length=18)

where will this be replaced? it doesn't match the configure_file template format